test/system/static.test.ts

```diff
@@ -88,8 +85,8 @@ test("static analysis builds the expected response", async (t) => {
   );
 
   t.deepEqual(
-    pluginResultWithSkopeoCopy.package.dependencies,
-    pluginResultWithDockerSave.package.dependencies,
+    pluginResultWithSkopeoCopy.scannedProjects[0].depTree.dependencies,
```

Comment:
In every test that we now use `scannedProjects`, could we also introduce an assertion that the length === 1? It will help us catch if there are more returned results! (At which point we can decide if that is expected)

Reply:
that's a bit like pouring cement over our code, because:
1. right now it's hard coded to return a single item
2. I'm starting to work on returning additional results
3. I have this assertion in a single place, duplicating it now means rewriting it in a few days.

when this area is stabilised a bit I'm happy to write more thorough tests

Reply:
> I have this assertion in a single place 

If this gives us enough confidence that it would cover the other cases then I'm happy with it! What I want to prevent here is for some unexpected reason we start creating more projects than we should 🙏

Reply:
that's a good concern and thanks for raising it :) but right now it's not needed.
hopefully the next piece of work (returning node dependencies) will solidify some of the tests